doc/classes/AudioStreamPlayer.xml

```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="AudioStreamPlayer" inherits="Node" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
+		Plays back audio.
 	</brief_description>
 	<description>
+		Plays background audio without dampening according to distance
```

Comment:
I don't think you need to go into what it *doesn't* do.  I don't think it's clear what "distance" refers to anyway.

Also, the sentence is missing a period.